pbxplore/test/test_regression.py

```diff
@@ -240,15 +251,10 @@ def test_xtc_input(self):
         status = exe.wait()
         print(out.decode('utf-8'))
         print(err.decode('utf-8'))
-        if IS_MDANALYSIS:
-            # MDanalysis is available, PBassign should run and produce the
-            # correct output
-            assert status == 0, 'PBassign exited with an error'
-            _assert_identical_files(os.path.join(REFDIR, output_fname),
-                                    os.path.join(out_run_dir, output_fname))
-        else:
-            # MDanalysis is not available, PBassign should fail
-            assert status != 0, 'PBassign shoud not have exited with a 0 code'
```

Comment:
On 18/05/16 15:15, Hub wrote:
>
> In pbxplore/test/test_regression.py 
> <https://github.com/pierrepo/PBxplore/pull/122#discussion_r63699546>:
>
> > @@ -240,15 +251,10 @@ def test_xtc_input(self):
> >          status = exe.wait()
> >          print(out.decode('utf-8'))
> >          print(err.decode('utf-8'))
> > -        if IS_MDANALYSIS:
> > -            # MDanalysis is available, PBassign should run and produce the
> > -            # correct output
> > -            assert status == 0, 'PBassign exited with an error'
> > -            _assert_identical_files(os.path.join(REFDIR, output_fname),
> > -                                    os.path.join(out_run_dir, output_fname))
> > -        else:
> > -            # MDanalysis is not available, PBassign should fail
> > -            assert status != 0, 'PBassign shoud not have exited with a 0 code'
>
> Indeed. I will reintroduce this one.
> One can argue we have to do the same for matplotlib & weblogo
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/pierrepo/PBxplore/pull/122/files/7d5d91e364c3a1b35edaa7e65b56a0b6df186b71#r63699546>
>
I did not dear increase the load ;)